src/common/cuepoint-manager.js

```diff
@@ -0,0 +1,72 @@
+//@flow
+import {Cue, FakeEvent} from '@playkit-js/playkit-js';
+
+interface CuePoint {
+  id: string;
+  startTime: number;
+  endTime: number;
+}
+
+export class CuePointManager {
+  Event = {
+    TIMED_METADATA_ADDED: 'timedmetadataadded'
+  };
+  Type = {
+    CuePointsTextTrack: 'CuePoints',
```

Comment:
enum keys are upper case - CUE_POINTS_TEXT_TRACK

Reply:
I think its better to separate the enum config and expose in from kaltura player index js (like playlist)